build.sbt

```
@@ -616,22 +622,26 @@ lazy val sqlDeltaImport = (project in file("sql-delta-import"))
       // `build/sbt "++ 2.11.12 publishLocal"` because Spark 3.0.0 and Delta 0.7.0 don't support
       // 2.11 and don't have the jars for Scala 2.11. In order to make our publish command pass, we
       // define Spark and Delta versions with 2.12 to make dependency resolution pass.
-      "org.apache.spark" % "spark-sql_2.12" % "3.0.0" % "provided",
-      "io.delta" % "delta-core_2.12" % "0.7.0" % "provided",
+      "io.netty" % "netty-buffer"  % "4.1.63.Final",
+      "org.apache.spark" % "spark-sql_2.13" % "3.2.0" % "provided",
```

```suggestion
      "org.apache.spark" %% "spark-sql" % "3.2.0" % "provided",
```
Could you also update the libraries below?